UPGRADE_GUIDE.md

```diff
@@ -6,6 +6,35 @@
 
 You must upgrade your project to 2019.3 to be able to use this version, and future versions, of the GDK for Unity. Any patch release of 2019.3 should work, but we test against 2019.3.7f1.
 
+### Unity Entities 0.7.0 upgrade
+
+We've upgraded our dependency on Unity's entities package from 0.1.0 to 0.7.0, this comes with a few breaking changes.
+
+You'll have to add the build-in module `com.unity.modules.assetbundle` to your project dependencies. This can be done through the package manager window.
```

Comment:
```suggestion
You'll have to add the built-in module `com.unity.modules.assetbundle` to your project dependencies. This can be done through the package manager window.
```